src/beginner-setup.md

```diff
@@ -0,0 +1,109 @@
+# Beginner Setup Guide: Running Sugar Activities Locally
+
+This guide will help beginners set up a local development environment for Sugar and run Sugar activities easily.
+
+---
+
+## 🧰 Prerequisites
+
+Before starting, make sure you have the following tools installed:
+
+### For Linux Users (Ubuntu/Debian-based):
+
+* `git`
+* `python3`
+* `pip`
+* `virtualenv`
+
+Install them using:
+
+```bash
+sudo apt update
```

Comment:
this is where the Linux version makes a difference. 'dnf' instead of 'apt' in the case of Fedora...